README-QUICKSTART.md

```diff
@@ -0,0 +1,152 @@
+# Scylla CDC Source Connector Quickstart
+
+## Synopsis
+
+This quickstart will show you how to setup the Scylla CDC Source Connector to replicate changes made in 
+a Scylla table using [Scylla CDC](https://docs.scylladb.com/using-scylla/cdc/cdc-intro/).
+
+## Scylla setup
+
+First, let's setup a Scylla cluster and create a CDC-enabled table.
+
+### Scylla installation
+
+For the purpose of this quickstart, we will configure a Scylla instance using Docker. You can skip this 
+section if you have already installed Scylla. To learn more about installing Scylla in production
+environments, please refer to the [Install Scylla page](https://docs.scylladb.com/getting-started/install_scylla/).
+
+1. Using [Docker](https://hub.docker.com/r/scylladb/scylla/), follow the instructions to launch Scylla.
+2. Start the Docker container, replacing the `--name` and `--host name` parameters with your own information. For example:
+   ```
+   docker run --name scylla-cdc-quickstart --hostname scylla-cdc-quickstart -d scylladb/scylla
+   ```
+3. Run `docker ps` to show the exposed ports. The output should be similar to this example:
+    ```
+    docker ps 
+    
+    CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS              PORTS                                                            NAMES
+    4fca02217055        scylladb/scylla     "/docker-entrypoint.…"   8 seconds ago       Up 7 seconds        22/tcp, 7000-7001/tcp, 9042/tcp, 9160/tcp, 9180/tcp, 10000/tcp   scylla-cdc-quickstart
+    ```
+   
+### Creating a CDC-enabled table
+
+Let's connect to your Scylla cluster and create a new CDC-enabled table. We will create an example table by 
+issuing the following CQL query and insert some example data:
+
+```
+CREATE KEYSPACE quickstart_keyspace WITH REPLICATION = {'class': 'SimpleStrategy', 'replication_factor': 1};
+
+CREATE TABLE quickstart_keyspace.orders(
+  customer_id int, 
+  order_id int, 
+  product text, 
+  PRIMARY KEY(customer_id, order_id)) WITH cdc = {'enabled': true};
+
+INSERT INTO quickstart_keyspace.orders(customer_id, order_id, product) VALUES (1, 1, 'pizza'); 
+INSERT INTO quickstart_keyspace.orders(customer_id, order_id, product) VALUES (1, 2, 'cookies');
+INSERT INTO quickstart_keyspace.orders(customer_id, order_id, product) VALUES (1, 3, 'tea');
+```
+
+If you already have a table you wish to use, but it does not have CDC enabled, you can turn it on by using the following CQL query:
+```
+ALTER TABLE keyspace.table_name with cdc = {'enabled': true};
+```
+
+To learn more about Scylla CDC, visit [Change Data Capture (CDC) page](https://docs.scylladb.com/using-scylla/cdc/).
+
+## Kafka setup
+
+Scylla CDC Source Connector works well with both [open-source Kafka](https://kafka.apache.org/) 
+and [Confluent Platform](https://www.confluent.io/). In this quickstart we will show how
```

Comment:
Maybe note here that the version of Kafka has to be high enough?

Reply:
Added.